roaringbitmap/src/main/java/org/roaringbitmap/buffer/MappeableBitmapContainer.java

```diff
@@ -1269,27 +1269,26 @@ public int nextSetBit(final int i) {
   }
 
   /**
-   * Find the index of the next unset bit greater or equal to i, returns -1 if none found.
+   * Find the index of the next clear bit greater or equal to i.
    *
    * @param i starting index
-   * @return index of the next unset bit
+   * @return index of the next clear bit
    */
-  public short nextUnsetBit(final int i) {
-    int x = i / 64;
+  public int nextClearBit(final int i) {
```

Comment:
Should this return an `int`? Was the decision to change this for efficiency, because you will typically immediately convert it back to an `int`?

Reply:
Symmetry, `nextSetBit` also returns an int.

BTW, AFAIK java lifts to ints on any operation, the only benefit short has over int is that arrays are stored denser.

Reply:
It does, but preserves sign when it does so, which would be the wrong thing to do here (and is the source of a lot of bugs in this library).